pkg/service/ipam-service.go

```diff
@@ -95,7 +98,11 @@ func (i *IPAMService) ListPrefixes(ctx context.Context, req *connect.Request[v1.
 
 	var result []*v1.Prefix
 	for _, cidr := range resp {
-		p := i.ipamer.PrefixFrom(ctx, cidr)
+		p, err := i.ipamer.PrefixFrom(ctx, cidr)
+		if err != nil {
+			i.log.Warn("skipping prefix of cidr", "error", err)
```

Comment:
Not related to this PR but probably it's a good idea to return an error to the client when this happens as otherwise it may works with wrong assumptions.